vnext/Shared/OInstance.cpp

```diff
@@ -545,13 +545,15 @@ std::vector<std::unique_ptr<NativeModule>> InstanceImpl::GetDefaultNativeModules
   std::vector<std::unique_ptr<NativeModule>> modules;
   auto transitionalProps{ReactPropertyBagHelper::CreatePropertyBag()};
 
+#if (defined(_MSC_VER) && !defined(WINRT))
```

Comment:
can you clarify this bit? why is the http module not available for uwp? (e.g. is it replaced by blob module?)

Reply:
The HTTP was being instantiated twice in MSRN (last one wins, though).
For a few reasons, including the fact that the Desktop variant does not have access to the property bag, ( @acoates-ms should know more details), Desktop needs to instantiate some modules here (HTTP, WebSocket, Timer...) for them to be available to the consuming app.

These "core" modules are instantiated somewhere else in MSRN:
CoreNativeModules.cpp.

In summary, this part of the change:
- Removes duplicate instantiations of core modules
- Ensures modules that require a property bag to communicate (HTTP, Blob, FileReader, WebSocket) use the appropriate bag instance (Desktop: OInstance local object. Universal: context's member).